python/amici/ode_export.py

```diff
@@ -2168,6 +2166,8 @@ def _write_index_files(self, name: str) -> None:
 
         for index, symbol in enumerate(symbols):
             symbol_name = strip_pysb(symbol)
+            if str(symbol) == '0':
```

Comment:
Why the string based comparison here? How sure are you this is always going to be 0 and non 0.0. Whats wrong with `symbol.is_zero`?

Reply:
Had the symbolic comparison first.
Turns out: Not all qunatities actually pass symbols, some (e.g., dwdx) pass strings! Then, symbolic comparison fails... Therefore the string.
Pretty sure, that `0` will do, as in all cases, which I might think of, the symbolic zeros are initialized by sympy.zeros(). This really creates a '0', whereas '0.0' may only happen if floats are involved. Anyhow, the only routines needing this currently is dtcldp.

However, I fully agree that these should *always* be symbols here. I'm happy to clean this up in the next PR. For the moment, these improvments really have to go into develop now...

Reply:
Yeah thats fine, `syms` should always yield symbols. fine to not address now, but please open an issue.

Reply:
Done, see #1061 .